tools/Google.Cloud.Tools.sln

```diff
@@ -14,8 +14,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Google.Cloud.Tools.VersionC
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Google.Cloud.Tools.VersionCompat.Tests", "Google.Cloud.Tools.VersionCompat.Tests\Google.Cloud.Tools.VersionCompat.Tests.csproj", "{7F116487-62FD-4F99-96B3-6CA428DD84E0}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Google.Cloud.Tools.SourceManipulation", "Google.Cloud.Tools.SourceManipulation\Google.Cloud.Tools.SourceManipulation.csproj", "{8C4505CA-44CB-425C-861D-227D486D5842}"
```

Comment:
May be we also need to add this project in Grafeas.V1.sln ?

Reply:
No, I've just copied the source file. Will look at the error more closely.

Reply:
(I'd done that locally, but the solution file hadn't saved...)